### PR TITLE
Update multi-geo-ediscovery-configuration.md

### PR DESCRIPTION
**Pull request by AdwoodMicrosoft**
Update per Sujit Narayanan to include Advanced eDiscovery capabilities and the limitation of the New-Compliance filter for GoLocals.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/multi-geo-ediscovery-configuration.md [hunk 1]
```diff
@@ -17,7 +17,9 @@ description: Learn how to use the Region parameter to configure eDiscovery for u
 
 # Microsoft 365 Multi-Geo eDiscovery configuration
 
-By default, an eDiscovery Manager or Administrator of a multi-geo tenant will be able to conduct eDiscovery only in the central location of that tenant. To support the ability to conduct eDiscovery for satellite locations, a new Compliance Security Filter parameter called "Region" is available via PowerShell.
+Advanced eDiscovery capabilities [a link](https://https://docs.microsoft.com/en-us/microsoft-365/compliance/overview-ediscovery-20?view=o365-worldwide) allow a Multi-Geo eDiscovery administrator to search all of the geos without needing to utilize a "Region" security filter. Data is exported to the Azure instance of the central location of the Multi-Geo tenant.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/multi-geo-ediscovery-configuration.md]:
```suggestion
[Advanced eDiscovery capabilities](https://docs.microsoft.com/microsoft-365/compliance/overview-ediscovery-20?view=o365-worldwide) allow a multi-geo eDiscovery administrator to search all of the geos without needing to utilize a "Region" security filter. Data is exported to the Azure instance of the central location of the multi-geo tenant. 
```

File: microsoft-365/enterprise/multi-geo-ediscovery-configuration.md [hunk 2]
```diff
@@ -17,7 +17,9 @@ description: Learn how to use the Region parameter to configure eDiscovery for u
...
+Advanced eDiscovery capabilities [a link](https://https://docs.microsoft.com/en-us/microsoft-365/compliance/overview-ediscovery-20?view=o365-worldwide) allow a Multi-Geo eDiscovery administrator to search all of the geos without needing to utilize a "Region" security filter. Data is exported to the Azure instance of the central location of the Multi-Geo tenant. 
+
+Without advanced eDiscovery capabilities, an eDiscovery Manager or Administrator of a multi-geo tenant will be able to conduct eDiscovery only in the central location of that tenant. To support the ability to conduct eDiscovery for satellite locations, a new Compliance Security Filter parameter called "Region" is available via PowerShell. This parameter can be used by tenants whose central location is in North America, Europe, or Asia Pacific. Advanced eDiscovery is recommended for tenant's whose central location is not in North America, Europe, or Asia Pacific and who need to perform eDiscovery across satellite geo locations.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/multi-geo-ediscovery-configuration.md]:
```suggestion
Without advanced eDiscovery capabilities, an eDiscovery manager or administrator of a multi-geo tenant will be able to conduct eDiscovery only in the central location of that tenant. To support the ability to conduct eDiscovery for satellite locations, a new compliance security filter parameter named "Region" is available via PowerShell. This parameter can be used by tenants whose central location is in North America, Europe, or Asia Pacific. Advanced eDiscovery is recommended for tenants whose central location is not in North America, Europe, or Asia Pacific and who need to perform eDiscovery across satellite geo locations. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@AdwoodMicrosoft Please implement the suggested changes. Thanks.

**Review by AdwoodMicrosoft — COMMENTED** [review]
Approved

**Pull request merged**